lib/additional_methods.rb

```diff
@@ -0,0 +1,24 @@
+
+class String
+  def without_quotes
+    fail("object should be string") unless self.class == String
+    quotes = ["\"", "'"]
```

Comment:
https://ruby-doc.org/core-2.4.0/String.html#method-i-tr
Spend some time to learn doc about system classes and their methods. Array, Hash, Numeric, String, Symbol and so on.